### ISSUE TITLE
Provider linter as npm module

### ISSUE THREAD

**Issue by RangerMauve** [lukeschafer/EmailLint#2]
It'd be nice if you put the linter module itself on NPM so that people could make more cool stuff with it.

**Comment by lukeschafer** [comment]
Yeah that'd be good. Someone promised to do the code split (site / module) but it appears they never did. I'll see if I can get to it.

**Comment by RangerMauve** [comment]
I forget where I saw it, but somebody wanted to use this, or something else, to create an email linter plugin for atom. If I find the issue again I'll post it here.

**Comment by coliff** [comment]
@RangerMauve That was me who requested that :-)  I came back here to suggest that to @lukeschafer 
https://github.com/sindresorhus/module-requests/issues/21#issuecomment-127278108
